src/broadcast/scrobblingmanager.cpp

```diff
@@ -0,0 +1,210 @@
+#include <QObject>
+
+#include "broadcast/scrobblingmanager.h"
+#include "broadcast/filelistener.h"
+#include "control/controlproxy.h"
+#include "engine/enginexfader.h"
+#include "mixer/deck.h"
+#include "mixer/playerinfo.h"
+#include "mixer/playermanager.h"
+
+ScrobblingManager::ScrobblingManager(PlayerManager *manager) :
+m_CPGuiTick("[Master]", "guiTick50ms",this),
```

Comment:
Some style nits: 
We use a new line and 4 or 8 space indent before the colon. All initializer are following in a column + 2 spaces. 
Opening brace is always at the line end. 
```
ScrobblingManager::ScrobblingManager(PlayerManager *manager) 
        : m_CPGuiTick("[Master]", "guiTick50ms",this),
          ...
          m_CPCrossfader("[Master]","crossfader", this) {
```